lesson_1/task_1.py

```diff
@@ -0,0 +1,23 @@
+# Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей. Дано a, b, c - стороны предполагаемого треугольника. Требуется сравнить длину каждого отрезка-стороны с суммой двух других. Если хотя бы в одном случае отрезок окажется больше суммы двух других, то треугольника с такими сторонами не существует. Отдельно сообщить является ли треугольник разносторонним, равнобедренным или равносторонним.г/* Треугольник существует только тогда, когда сумма любых двух его сторон больше третьей. Дано a, b, c - стороны предполагаемого треугольника. Требуется сравнить длину каждого отрезка-стороны с суммой двух других. Если хотя бы в одном случае отрезок окажется больше суммы двух других, то треугольника с такими сторонами не существует. Отдельно сообщить является ли треугольник разносторонним, равнобедренным или равносторонним.
+def existence_of_triangle(a,b,c):
+    if (a + b > c) and (b + c > a) and  (a + c > b):
+        return True
+    else:
+        return False
+def type_of_triangle(a,b,c):
+    if a == b == c:
+        return "равносторонний."
+    if a == b or b == c or c == a:
+        return "равнобедренный."
+    return "не обладает свойствами равенства сторон, разносторонний."
```

Comment:
Согласно концепции повторного использования кода, не принято именовать блок кода, то есть определять функцию, и если она планируется вызываться единожды.

Reply:
Кроме того, согласно документации PEP8 функций должны иметь 2 пустые строки снизу и сверху от своего определения.
```suggestion
def existence_of_triangle(a, b, c):
    if (a + b > c) and (b + c > a) and (a + c > b):
        return True
    else:
        return False


def type_of_triangle(a, b, c):
    if a == b == c:
        return "равносторонний."
    if a == b or b == c or c == a:
        return "равнобедренный."
    return "не обладает свойствами равенства сторон, разносторонний."
```